doc/releases/changelog-dev.md

```diff
@@ -3,6 +3,8 @@
 # Release 0.25.0-dev (development release)
 
 <h3>New features since last release</h3>
+* Fixed error of IsingXY gate definition in latex.
```

Comment:
This should also go in the bug-fixes section of the changelog, have keep a space between the heading and the bullets.

```suggestion
* Updated IsingXY gate doc-string.
```

Reply:
Done!